script.js

```diff
@@ -0,0 +1,98 @@
+class Books {
+  constructor() {
+    this.BookList = {};
+    this.DisplayBooks = document.querySelector('.collection');
+    this.NewTitle = document.querySelector('#title');
+    this.NewAuthor = document.querySelector('#author');
+    this.SaveBooks = localStorage.getItem('BookList');
+    this.AddBook = document.querySelector('#btn');
+    this.ErrorChecker = document.querySelector('.checker');
+    this.StoreBooks = [];
+    this.DecideOnDisplay();
+  }
+
+  ClearFields = () => {
+    this.NewTitle.value = '';
+    this.NewAuthor.value = '';
+  };
+
+  Validate() {
+    if (this.NewTitle.value !== '' && this.NewAuthor.value !== '') {
+      this.ErrorChecker.textContent = '';
+      return true;
+    }
+  }
+
+  DisplayAllBooks = () => {
+    this.DisplayBooks.innerHTML = '';
+    this.HoldBooks = JSON.parse(localStorage.getItem('BookList'));
+    if (this.HoldBooks != null) {
+      this.StoreBooks = this.HoldBooks;
+      this.HoldBooks.forEach((Books, index) => {
+        this.DisplayBooks.innerHTML += `
+         <tr class="Display" >
+         <td><span>"</span>${Books.Title}<span>" by </span>${Books.Author}</td>
+         <td><button class="remove" id="${index}">Remove</button></td>
+         </tr>`;
+      });
+    }
+  };
+
+  DecideOnDisplay = () => {
+    if (this.DisplayBooks.innerHTML === '') {
+      this.HoldBooks = JSON.parse(localStorage.getItem('BookList'));
+      this.StoreBooks = this.HoldBooks;
+      this.HoldBooks.forEach((Books, index) => {
+        this.DisplayBooks.innerHTML += `
+         <tr class="Display">
+         <td><span>"</span>${Books.Title}<span>" by </span>${Books.Author}</td>
+         <td><button class="remove" id="${index}">Remove</button></td>
+         </tr>`;
+      });
+    } else {
+      this.DisplayAllBooks();
+    }
+  };
+
+  AddNewBook = () => {
```

Comment:
- [ ] The previous reviewer noted that there are some issues with the 'Add' button. When attempting to add a new book, the input field remains unchanged and it is possible to detect errors in the console. Please review the attached video for more information.

https://user-images.githubusercontent.com/69011963/212045356-763f2927-180e-4954-aafb-9c1c834dd618.mov

Reply:
@Rustamxon7 Hello,

Please, we need to know what exactly is missing from our design that shouldn't be.
Also, kindly help us with more info on the 'add' button. What exactly can we do?
This is our last review request, we really want to make it count 😃 

Your assistance will be highly appreciated
Thanks

Reply:
Hello @Uthmanbello, 
I got your message. I'm here to help

Reply:
@Uthmanbello, @alexander16108 Could you write me via slack?

Reply:
Since it was hard time to follow up the conversation here I left my suggestion in slack @Uthmanbello. Kindly check it out. Remember it's not 100% efficient you need to work front side and error handling side of your code.